packaging/nnstreamer.spec

```diff
@@ -899,7 +899,15 @@ export NNSTREAMER_CONVERTERS=${NNSTREAMER_BUILD_ROOT_PATH}/ext/nnstreamer/tensor
     popd
 
 python3 tools/development/count_test_cases.py build tests/summary.txt
-%endif #if unit_test
+%else
+%if 0%{?edge_test}
+    bash %{test_script} tests/nnstreamer_edge
```

Comment:
add tests/nnstreamer_query

Reply:
This is directory path.
`unittest_query` and `unittest_edge` execution files are in tests/nnstreaer_edge.
What I don't know is that ssat doesn't run `tests/nnstreamer_edge/query/runTesh.sh` and `tests/nnstreamer_edge/edge/runTesh.sh`. I think it should be run..

Reply:
ok :)